### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by shmuga**
PR for improvements in testing and so on.
See commit messages for more information.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -9,6 +9,9 @@
     "no-console": 0,
     "func-names": 0,
     "no-param-reassign": 0,
-    "prefer-template": 0
+    "prefer-template": 0,
+    "import/no-extraneous-dependencies": ["error",
+      {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}
```
Comment (dchertousov) [inline .eslintrc]: Didn't know about this trick :)

File: test/Utils/UtilsValidator.test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import uAPI from '../../src';
+import UtilsValidator from '../../src/Services/Utils/UtilsValidator';
+
+describe('#UtilsValidator', () => {
+  describe('.CURRENCY_CONVERSION', () => {
+    it('should throw error for undefined', () => {
+      const fn = () => UtilsValidator.CURRENCY_CONVERSION({});
+      expect(fn).to.throw(uAPI.errors.Utils.UtilsValidationError.CurrenciesMissing);
```
Comment (dchertousov) [inline test/Utils/UtilsValidator.test.js]: I see, you finally liked expect API :)

File: test/Utils/UtilsValidator.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      const fn = () =>
+        UtilsValidator.CURRENCY_CONVERSION({ currencies: [{}] });
+      expect(fn).to.throw(uAPI.errors.Utils.UtilsValidationError.CurrenciesMissing);
+    });
+
+    it('should correct validate and return object', () => {
+      const params = UtilsValidator.CURRENCY_CONVERSION(
+        { currencies: [{ from: 'UAR', to: 'EUR'}] }
+      );
+      expect(params).not.equal(null);
```
Comment (dchertousov) [inline test/Utils/UtilsValidator.test.js]: Why checking for null?

Comment (shmuga) [inline test/Utils/UtilsValidator.test.js]: Trying to find that params are returned

File: test/Utils/UtilsValidator.test.js (hunk 2 above)
Comment (shmuga) [inline test/Utils/UtilsValidator.test.js]: Nope, but i like common way to do something 😄

File: test/Utils/UtilsValidator.test.js (hunk 3 above)
Comment (dchertousov) [inline test/Utils/UtilsValidator.test.js]: Please, check what is returned, to check structure.

**Pull request merged**